sport/app/football/controllers/WallchartController.scala

```diff
@@ -77,6 +78,41 @@ class WallchartController(
         .getOrElse(NotFound)
     }
 
+  def renderIndividualGroupTableEmbed(competitionTag: String, groupId: String): Action[AnyContent] =
+    Action { implicit request =>
+      competitionsService
+        .competitionsWithTag(competitionTag)
+        .map { competition =>
+          val page = new FootballPage(
+            competition.url.stripSuffix("/"),
+            "football",
+            s"${competition.fullName} group tables",
+          )
+          val competitionStages = new CompetitionStage(competitionsService.competitions)
+            .stagesFromCompetition(competition, KnockoutSpider.orderings)
+
+          val groupStages = competitionStages.collectFirst { case stage: Groups => stage }
+
+          groupStages match {
```

Comment:
Nitpick, feel free to ignore - I think this could be:

```scala
groupStages.map(group => ...).getOrElse(NotFound)
```

Reply:
Yes that's I wanted to do at first but couldn't make it work. Thanks for showing it